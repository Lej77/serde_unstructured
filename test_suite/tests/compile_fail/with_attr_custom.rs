
#[derive(serde_unstructured::SerdeView)]
struct Data {
    #[serde_view(with = "path")]
    field: u8,
}

fn main() {}