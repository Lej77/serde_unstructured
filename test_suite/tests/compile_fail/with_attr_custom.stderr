error: SerdeView can't project into fields that are using custom deserialization, remove any `#[serde_view(with = "module")]` or #[serde_view(deserialize_with = "path")] attribute.
 --> tests/compile_fail/with_attr_custom.rs:5:5
  |
5 |     field: u8,
  |     ^^^^^
